scorched/response.py

```diff
@@ -108,9 +108,7 @@ def from_json(cls, node, datefields=()):
     def _prepare_docs(self, docs, datefields):
         for doc in docs:
             for name, value in list(doc.items()):
-                if name in datefields:
```

Comment:
True. But this check was moved, not removed. See above first lines of is_datetime_field (lines 136, 137)